### DISCUSSION TITLE
Custom Entity Extract Prompt for GraphRag or LightRag

### DISCUSSION THREAD

**Discussion by eddprogrammer** [Cinnamon/kotaemon#567] [Q&A]
Hi, how do I change the default entity extraction prompts used by GraphRag or LightRag? If I am using them directly, I would change the prompt.py in their respectively library directory directly. Not sure how to do that when they are bundled into Kotaemon.

**Comment by taprosoft** [comment]
Done in https://github.com/Cinnamon/kotaemon/releases/tag/v0.9.8
